### ISSUE TITLE
Hide terminating namespaces in dropdown

### ISSUE THREAD

**Issue by obeyler** [kubeapps/kubeapps#543]
Kubeapps deployed with chart helm 0.3.0

namespace ( test 1-ba...) doesn't belong to my kubernetes cluster so I supposed It's a bug  
 
![image](https://user-images.githubusercontent.com/23224162/44584308-cc602f00-a7a8-11e8-94b7-fd300760c69c.png)

**Comment by obeyler** [comment]
you right I see them with command line 
```
 kubectl get namespaces   
NAME                                        STATUS        AGE
default                                     Active        10d
kubeapps                                    Active        2d
stash                                       Active        8d
test-1ba341e8-fdc1-404d-82cf-f27f1f35f003   Terminating   3h
test-1c621dee-78cc-4141-b5c7-dad2ce69b5ad   Terminating   3h
test-9736d218-4a77-4926-9d6a-cbdd0c3d3b5f   Terminating   3h
```
I was surprised because I don't see them into the Kubernetes dashboard

They are on status "teminating" so I don't know if they should be present into your list

**Comment by andresmgot** [comment]
Yes, we can probably just list the namespaces that are active. What do you think @prydonius?

**Comment by migmartri** [comment]
@obeyler thanks for the report!

@andresmgot. I sounds reasonable to me to filter the list of namespaces to show only active ones

**Comment by prydonius** [comment]
The namespaces should eventually go away if they are in the terminating state, but we could indeed filter by the deleted timestamp.

**Comment by migmartri** [comment]
@obeyler are those namespace stuck into terminating state?

**Comment by obeyler** [comment]
yes they are :-( And I try every workaround found on internet to eliminate them without any sucess

**Comment by prydonius** [comment]
Okay I think the action item here is to filter the terminating namespaces and only show active ones.

**Comment by obeyler** [comment]
I agree

**Comment by andresmgot** [comment]
this has been fixed as part of #2018
